.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -202,6 +204,13 @@ if [ "$COMMITTED" == "true" ] && [ "$COMMAND" == "downstream" ] && [ "$CHANGELOG
     git commit --signoff --amend --no-edit
 fi
 
+if [ "$COMMITTED" == "true" ] && [ "$COMMAND" == "downstream" ] && [ "$COPYRIGHT" == "true" ]; then
+    # Add copyright headers using copywrite CLI
+    copywrite headers --spdx 'MPL-2.0'
+    git add .
```

Comment:
We could use output from the copywrite tool to specifically only stage the files it has edited

Reply:
The CLI has a `--plan` flag that could be useful, but the `headers` command outputs affected files anyway